robusta_krr/core/integrations/kubernetes.py

```diff
@@ -129,7 +129,7 @@ async def _list_workflows(
             self.debug(f"Skipping {kind}s in {self.cluster}")
             return
 
-        if kind == "rollout" and not self.__rollouts_available:
+        if kind == "Rollout" and not self.__rollouts_available:
```

Comment:
any reason why you changed the casing?

Reply:
To be used in object kind (previously they were taken from API response)